### ISSUE TITLE
Labels are flickering

### ISSUE THREAD

**Issue by xsellier** [binogure-studio/city-game-studio-i18n#103]
Hardware specs:
```
windows 7 edition familial prénium
i3 CPU  M370   2,4GHz
4 Go ram
carte graphique , Nvidia GForce GTX 1050
```

Screenshot:
![image](https://user-images.githubusercontent.com/6616005/53751477-5422a680-3e7a-11e9-9632-824e53f14377.png)

Appears to any resoluion (even fullscreen)

**Comment by xsellier** [comment]
![image](https://user-images.githubusercontent.com/6616005/53751814-2427d300-3e7b-11e9-8b59-5219b6942cca.png)

**Comment by xsellier** [comment]
Related issue:
https://github.com/godotengine/godot/issues/9913

**Comment by xsellier** [comment]
Fixed in v0.26.0
